cdshealpix/tests/test_nested_healpix.py

```diff
@@ -61,6 +61,15 @@ def test_healpix_to_lonlat_on_brocasted_arrays():
     b = np.broadcast_to(3, (3, 4, 5))
     healpix_to_lonlat(ipix=b, depth=depth)
 
+def test_healpix_to_lonlat_on_broadcasted_arrays2():
+    level = np.arange(2)
+    nside = 2**level
+
+    ipix = np.arange(3)
+
+    lon, lat = healpix_to_lonlat(ipix[:, np.newaxis], level[np.newaxis, :])
+    assert(lon.shape == lat.shape and lon.shape == (3, 2))
```

Comment:
assert is not a function. The usual coding style in Python is to not write `()`, suggest to remove it.

It's not really a problem usually.

But it can lead to cases like `assert(False, False)` that doesn't raise `AssertionError`, because it's creating a 2-tuple, which is truthy:
```
>>> bool((False, False)) 
True
```